### ISSUE TITLE
Remove the default value of container's log_driver and set `Computed: true`

### ISSUE THREAD

**Issue by suzuki-shunsuke** [terraform-providers/terraform-provider-docker#270]
### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

`v0.12.26`

### Affected Resource(s)

- docker_container

### Terraform Configuration Files
```hcl
provider "docker" {
  version = "2.7.0"
}

resource "docker_image" "nginx" {
	name = "nginx"
}

resource "docker_container" "nginx" {
  image    = docker_image.nginx.latest
  name     = "nginx"
  restart  = "on-failure"
  must_run = "true"
}
```

### Expected Behavior

When `log-driver` is set to `none` in the Docker daemon's configuration `daemon.json`,
created container's log driver should be `none`.

https://docs.docker.com/config/containers/logging/configure/#configure-the-default-logging-driver

daemon.json

```json
{
  "log-driver": "none"
}
```

### Actual Behavior

Created container's log driver is `json-file` bacause `log_driver`'s default value is `json-file`.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

```
$ terraform apply
$ docker inspect nginx # check LogConfig.
```

### How to fix

Remove the default value of container's `log_driver` and set `Computed: true`.

**Comment by suzuki-shunsuke** [comment]
I have created a pull request to fix this issue.
https://github.com/terraform-providers/terraform-provider-docker/pull/269

**Comment by mavogel** [comment]
Closed by #269
